hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/subscription/consumergroup/ConsumerGroupAlreadyScheduledToDeleteException.java

```diff
@@ -0,0 +1,24 @@
+package pl.allegro.tech.hermes.management.domain.subscription.consumergroup;
+
+import static java.lang.String.format;
+import static pl.allegro.tech.hermes.api.ErrorCode.OTHER;
+
+import pl.allegro.tech.hermes.api.ErrorCode;
+import pl.allegro.tech.hermes.management.domain.ManagementException;
+
+public class ConsumerGroupAlreadyScheduledToDeleteException extends ManagementException {
```

Comment:
since this exception is always ignored and will never return 5xx to the user, perhaps we should simply extend RuntimeException instead of `ManagementException`?

Reply:
sure